app/services/strategy_scanning_policy_service.py

```
@@ -0,0 +1,248 @@
+                            "strategy_key": policy.strategy_key,
+                            "max_symbols": policy.max_symbols,
+                            "chunk_size": policy.chunk_size,
+                            "priority": policy.priority,
+                            "enabled": bool(policy.enabled),
+                            "created_at": policy.created_at,
+                            "updated_at": policy.updated_at,
+                        }
+            except SQLAlchemyError as error:
+                self.logger.error(
+                    "Failed to load strategy scanning policies", error=str(error)
+                )
```

Ruff (0.14.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)